src/scaffoldmaker/meshtypes/meshtype_3d_wholebody2.py

```diff
@@ -1,199 +1,675 @@
+        options["Foot height"] = 1.0
+        options["Foot length"] = 2.5
+        options["Foot thickness"] = 0.3
+        options["Foot width"] = 1.0
+        options["Inner proportion default"] = 0.7
+        options["Inner proportion head"] = 0.35
+        return options
+
+    @classmethod
+    def getOrderedOptionNames(cls):
+        return [
+            "Head depth",
+            "Head length",
+            "Head width",
+            "Neck length",
+            "Shoulder drop",
+            "Shoulder width",
+            "Arm lateral angle degrees",
+            "Arm length",
+            "Arm top diameter",
+            "Wrist thickness",
+            "Wrist width",
+            "Hand length",
+            "Hand thickness",
+            "Hand width",
+            "Thorax length",
+            "Abdomen length",
+            "Torso depth",
+            "Torso width",
+            "Pelvis drop",
+            "Pelvis width",
+            "Leg lateral angle degrees",
+            "Leg length",
+            "Leg top diameter",
+            "Leg bottom diameter",
+            "Foot height",
+            "Foot length",
+            "Foot thickness",
+            "Foot width",
+            "Inner proportion default",
+            "Inner proportion head"
+        ]
+
+    @classmethod
+    def checkOptions(cls, options):
+        dependentChanges = False
+        for key in [
+            "Head depth",
+            "Head length",
+            "Head width",
+            "Neck length",
+            "Shoulder drop",
+            "Shoulder width",
+            "Arm length",
+            "Arm top diameter",
+            "Wrist thickness",
+            "Wrist width",
+            "Hand length",
+            "Hand thickness",
+            "Hand width",
+            "Pelvis drop",
+            "Pelvis width",
+            "Thorax length",
+            "Abdomen length",
+            "Torso depth",
+            "Torso width",
+            "Leg length",
+            "Leg top diameter",
+            "Leg bottom diameter",
+            "Foot height",
+            "Foot length",
+            "Foot thickness",
+            "Foot width"
+        ]:
+            if options[key] < 0.1:
+                options[key] = 0.1
+        for key in [
+            "Inner proportion default",
+            "Inner proportion head"
+        ]:
+            if options[key] < 0.1:
+                options[key] = 0.1
+            elif options[key] > 0.9:
+                options[key] = 0.9
+        for key in [
+            "Arm lateral angle degrees"
+        ]:
+            if options[key] < -60.0:
+                options[key] = -60.0
+            elif options[key] > 200.0:
+                options[key] = 200.0
+        for key in [
+            "Leg lateral angle degrees"
+        ]:
+            if options[key] < -20.0:
+                options[key] = -20.0
+            elif options[key] > 60.0:
+                options[key] = 60.0
```

Comment:
I've noticed the bifurcation junction that forms the hip/pelvis joining to the leg segments don't generate properly once the angle is greater than ~50 deg. Maybe it's best to keep the limit to around 40 deg?

Reply:
As above, not limiting to just ranges that work.